srcpkgs/wlroots/template

```diff
@@ -1,7 +1,6 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-reverts="0.6.0_1"
-version=0.5.0
+version=0.6.0
 revision=2
```

Comment:
revision=1

Reply:
but we already had a revision 1? wlroots was bumped then reverted.

Reply:
see @jnbr comment above

Reply:
My fault, didn't look at `reverts` :-)